client/reader/discover/site-attribution.jsx

```diff
@@ -58,13 +60,13 @@ class DiscoverSiteAttribution extends React.Component {
 						{ translate( 'visit' ) } <em>{ attribution.blog_name }</em>
 					</a>
 				</span>
-				{ !! this.props.followUrl
-					? <FollowButton
-							siteUrl={ this.props.followUrl }
-							iconSize={ 20 }
-							onFollowToggle={ this.onFollowToggle }
-						/>
-					: null }
+				{ !! this.props.followUrl ? (
```

Comment:
@samouri hi! Why just not use ```&&``` and avoid ternary?

```javascript
{ !! this.props.followUrl && (
					<FollowButton
						siteUrl={ this.props.followUrl }
						iconSize={ 20 }
						onFollowToggle={ this.onFollowToggle }
					/>
				) }
```

Reply:
hi @danielnass! These changes are the result of a codemod. I'd rather like to avoid changing what the codemod does in this PR, but we could simplify in another PR. I prefer this because codemod changes tend to be very safe, while hand-made changes add a new and exciting element of human error and make the review quite a bit more work.

Oh how I yearn for typescript / flow. :)

Reply:
@blowery thanks for the explanation 😄